css-module-bikeshed/Overview.bs

```diff
@@ -159,3 +159,7 @@ Internal display model: the 'foo' property {#the-foo-property}
 			DOMString baz = "qux";
 		};
 	</pre>
+
+<h2 class=no-num id=priv-sec>Privacy and Security Considerations</h2>
+
+Make some considerations about privacy and security.
```

Comment:
Well, adding this to the template only serves to silence a Bikeshed warning telling you you need to write this part. If we replace the warning with an issue in the spec, that's fine, but there was a reason for that warning: this is a part of the spec that is at risk of being forgotten unless you are explicitly reminded about.